### DISCUSSION TITLE
I dont have access to github copilot... How to re-generate my code? How to re-authorise?

### DISCUSSION THREAD

**Discussion by BlueKiwi94** [community/community#14412] [Copilot Conversations] [answered]
Hi,

I might have signed in to my co-pilot with a wrong github account and now I am locked out. I cannot find any information on how to re-authorise (Visual Studio 22). I tried re-installing my extension, disabling co-pilot app on the other github account and also signing out of github in visual studio 22. Nothing works, I can only see the sad output:

GitHub Copilot 1.10.0.3 (v1.10.0.3@764d4f318)
GitHub Copilot Agent v1.10.0
You don't have access to GitHub Copilot.

**Comment by BlueKiwi94** [comment]
It looks completely messed up because even when I click on the "sign in" option in the extension window (icon in the bottom left) I am told "you dont have access..."

**Comment by BlueKiwi94** [comment, marked as answer]
nvm figured it out you need to delete the files at C:\Users\%USERNAME%\appdata\local\github-copilot

**Reply by ghost** [reply]
Any way for linux?

**Reply by BlueKiwi94** [reply]
Visual Studio is not available on Linux? 👀

**Reply by ghost** [reply]
Oh, I'm running VSCodium

**Reply by akinolu52** [reply]
what of mac?

**Reply by bortvern** [reply]
Thank you kind human, this worked for me as well.

**Reply by rodrigobentancor** [reply]
Worked like a charm, thanks!
